content/docs/5.1/concepts/explore-tikv-features/cas.md

```diff
@@ -0,0 +1,144 @@
+---
+title: CAS on RawKV
+description: Compare And Swap
+menu:
+    "5.1":
+        parent: Features
+        weight: 4
+---
+
+This page walks you through a simple demonstration of how to use TiKV's `compare-and-swap`.
+
+In RawKV, `compare-and-swap (CAS)` is an atomic instruction used in multi-threading to achieve synchronization. It compares the `current_value` of a `key` with a given value and, only if they are the same, modifies the contents of that `key` to a new given value. This is done as a single atomic operation. The atomicity guarantees that the new value is calculated based on up-to-date information; if the value had been updated by another thread in the meantime, the write would fail.
+
+## Prerequisites
```

Comment:
no need to install tiup and start the tikv cluster again, since it already expressed in https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/

Let's leave the content to how to use CAS in Java and Rust